cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -1452,6 +1452,19 @@ protected HttpResponse setProperties(String id, NamespaceMeta meta) throws Excep
                  GSON.toJson(meta));
   }
 
+  protected HttpResponse setRepository(String id, String repoString) throws Exception {
```

Comment:
should also have a method that takes a RepositoryConfig object instead of a string